### PR TITLE
CollapsibleSection refactor

### PR DESCRIPTION
**Pull request by Erin007**
Separate props to pass individually rather than in SectionData blob.

### CODE REVIEW

**Comment by Erin007** [comment]
This is the last of this refactors to separate out props.

**Comment by Erin007** [comment]
cc: @tanyaparker @joshlory

File: apps/src/templates/teacherHomepage/CollapsibleSection.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+  renderContent() {
+    const content = this.props.children;
+
+    if (this.state.open) {
+      if (content.length > 1) {
+        return (
+          <div style={styles.content}>
+             {content.slice(0,2).map((course, index) =>
+               <div style={styles.card} key={index}>
```
Comment (Bjvanminnen) [inline apps/src/templates/teacherHomepage/CollapsibleSection.jsx]: styles.card doesn't appear to exist?

File: apps/src/templates/teacherHomepage/CollapsibleSection.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+             )}
+          </div>
+        );
+      }
+      return (
+        <div style={styles.content}>
+          {content}
+        </div>
+      );
+    }
```
Comment (Bjvanminnen) [inline apps/src/templates/teacherHomepage/CollapsibleSection.jsx]: Suggestion:
You first figure out the set of items you want to display, and make it an array.
```jsx
const childItems = _.isArray(this.props.children) ? this.props.children.slice(0, 2) : [this.props.children];
```
Now I think you can have a much simpler return
```jsx
return (
  <div style={styles.content}>
    {childElements.map((course, index) => 
      <div style={styles.card} key={index}>
        {course}
      </div>
    }
  </div>
);

File: apps/src/templates/teacherHomepage/CollapsibleSection.story.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  link: "link to the course",
+  assignedSections: []
+};
+
+export default storybook => {
+  return storybook
+    .storiesOf('CollapsibleSections', module)
+    .addStoryTable([
+      {
+        name: 'Recent Courses - 2 courses ',
```
Comment (Bjvanminnen) [inline apps/src/templates/teacherHomepage/CollapsibleSection.story.jsx:19]: Worth adding a story for 3 courses, and making sure that those beyond the first arent rendered?

Comment (Erin007) [reply apps/src/templates/teacherHomepage/CollapsibleSection.story.jsx:19]: ✅

**Pull request merged**